templates/image-builder.yml

```diff
@@ -43,7 +43,7 @@ objects:
         - image: "${IMAGE_NAME}:${IMAGE_TAG}"
           name: image-builder
           ports:
-          - containerPort: "${BACKEND_LISTENER_PORT}"
+          - containerPort: 8086
```

Comment:
Wait, without quotes?

Reply:
@Gundersanne indeed. Couldn't get it to work and the docs said to do this. 🤷🏻‍♂️